projects/demo/src/modules/components/sheet-dialog/sheet-dialog.template.html

```diff
@@ -57,11 +57,7 @@
             </button>
             <ng-template #template>
                 <div class="label">
-                    <tui-avatar
-                        size="m"
-                        text="Karl Gambolputty"
-                        [rounded]="true"
-                    ></tui-avatar>
+                    <tui-avatar size="m">Karl Gambolputty</tui-avatar>
```

Comment:
This one would overflow. One way to handle it, other than initials, is `<span tuiFade>...</span>`

Reply:
fixed